README.md

```diff
@@ -67,6 +67,33 @@ You can also perform a request to the Bot API while [sending an answer to the we
 override def onMessageReply(msg: Message): F[Option[Method[_]]] =
   Sync[F].pure(Some(sendMessage(chatId = ChatIntId(msg.chat.id), text = "Hello, world!")))
 ```
+In addition, the library provides a way to compose multiple Webhook
+bots into a sigle `Http4s` Server that will handle the webhooks
+registration as well as the incoming requests of all the composed
+bots. You ultimately decide at which host:port the server will be
+binded to:
+
+``` scala
+import scala.concurrent.ExecutionContext.Implicits.global
+
+val api1: Api[IO] = BotApi(http, baseUrl = s"https://api.telegram.org/bot$bot_token1")
+val api2: Api[IO] = BotApi(http, baseUrl = s"https://api.telegram.org/bot$bot_token1")
+val bot1: MyWebhookbot = new MyWebhookbot[IO](api1, 80, "ServerVisibleFromOutside", s"/$bot_token1")
+val bot2: MyWebhookbot = new MyWebhookbot[IO](api2, 3000, "ServerVisibleFromOutside", s"/$bot_token2")
+
+WebhookBot.compose[IO](
+    List(bot1, bot2),
+    8080,
+    ExecutionContext.global, //optional, global as default
+    "127.0.0.1" //optional, localhost as default
+  ).useForever.runSyncUnsafe()
+```
+
+Keep in mind that the ports and hosts used in the creation of each bot
+will be not used when composed. The pair host plus port passed as input to
```

Comment:
>Keep in mind that the ports and hosts used in the creation of each bot
will be not used when composed.

Well, this is confusing. We can move the host/port parameters to the WebhookBot's `start` method and deprecate its primary constructor.

Reply:
I did the change, but there is a bad news: I can deprecate the constructor otherwise I end up facing this error

```
bad, multiple overloaded alternatives of WebhookBot constructor define default arguments.
```

and it can't be done without breaking the backward compatibility. A way could be to remove the `abstract` modifier and have an apply method in the companion object, but then the user could instantiate the `WebhookBot` directly.
I decided to just deprecate the parameters and the old `start` method adding a proper documentation.
Let me know if you have a better alternative.
Cheers

Reply:
Let's just move the parameters and make a major release then.

Reply:
Ok, so I revert the changes about deprecation :wink: